treesit-auto.el

```diff
@@ -132,6 +132,11 @@ automatic installation (or prompting, based on the value of
       :ts-mode 'css-ts-mode
       :remap 'css-mode
       :url "https://github.com/tree-sitter/tree-sitter-css")
+    ,(make-treesit-auto-recipe
+      :lang 'dart
+      :ts-mode 'dart-ts-mode
+      :remap 'dart-mode
+      :url "https://github.com/ast-grep/tree-sitter-dart")
```

Comment:
In keeping with the most recent changes on the main branch, we have a new `:ext` keyword, and I'm pretty sure dart uses this:

```suggestion
      :url "https://github.com/ast-grep/tree-sitter-dart"
      :ext "\\.dart\\'")
```

Does that work for you?  Are there any other extensions to consider here?